pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -30,8 +30,12 @@ func ClusterChecksTypes() []status.ConditionType {
 type GenevaLoggingSpec struct {
 	// +kubebuilder:validation:Pattern:=`[0-9]+.[0-9]+`
 	ConfigVersion string `json:"configVersion,omitempty"`
-	// +kubebuilder:validation:Enum=DiagnosticsProd;Test
```

Comment:
@troy0820 - "I'm not sure why this is removed is the kubebuilder:validation:Enum=DiagnosticsProd,Test;CaFairFax is also on line 35"

Reply:
I'm not sure either - may just be a caveat of how I copied and pasted to create the new parameters. The appropriate kubebuilder tags are there though.

Reply:
It is bellow : `// +kubebuilder:validation:Enum=DiagnosticsProd;Test;CaFairfax`. So change expected